ganeti/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG CENTOS_VER="systemd"
+FROM osuosl/centos:${CENTOS_VER}
```

Comment:
`systemd` centos version?

Reply:
This uses the [centos/systemd](https://hub.docker.com/r/centos/systemd/) container so you can run a container using systemd.